website/source/layouts/google.erb

```diff
@@ -60,6 +60,9 @@
 			<li<%= sidebar_current("docs-google-resource-dns-record-set") %>>
 			<a href="/docs/providers/google/r/dns_record_set.html">google_dns_record_set</a>
 			</li>
+			<li<%= sidebar_current("docs-google-resource-storage-bucket") %>>
+			<a href="/docs/providers/google/r/storage_bucket.html">google_dns_record_set</a>
```

Comment:
Seems like the link name was not updated from copy-paste, probably should be st. like `google_storage_bucket` instead of `google_dns_record_set`.